src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -135,7 +135,7 @@ class MediaController(override val config: TableauxConfig,
       _ <- {
         import FutureUtils._
         promisify({ p: Promise[Unit] =>
-          vertx.fileSystem.delete(path.toString(), {result => p.success()})
+          vertx.fileSystem.delete(path.toString(), {result => p.success(())})
```

Comment:
What happens if fileSystem.delete fails? Shouldn't it check the result and make a p.success()/p.failure()?